src/PortingAssistant.Client.Porting/PortingProjectFile/PortingProjectFileHandler.cs

```diff
@@ -70,10 +75,10 @@ public PortingProjectFileHandler(ILogger<PortingProjectFileHandler> logger)
             }
             catch (Exception ex)
             {
-                _logger.LogError($"Failed to port projects{projectPaths} with error: {ex}");
+                _logger.LogError($"Failed to port solution {solutionPath} with error: {ex}");

```

Comment:
What if we're porting one (or more) projects instead of the whole solution? How can we associate this to one project?

Reply:
same above